### PR TITLE
update the alert message to read better

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation

https://issues.redhat.com/browse/INTLY-10170

## What

Change the message in two alerts 

## Why

for readability 

## How

update messages in deploy/template/prometheus-rule.yaml

## Verification Steps
TODO

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [ ] Finished task
- [ ] TODO

### CODE REVIEW

File: deploy/template/prometheus-rule.yaml [hunk 1]
```diff
@@ -95,7 +95,7 @@ spec:
         - alert: KeycloakAPIRequestDuration90PercThresholdExceeded
           annotations:
             message: >-
-              Less than 90% of the RH SSO API endpoints in namespace {{ $labels.namespace }} are taking longer than 1s for the last 5 minutes.
+              90% of the total requests are not served within 10 seconds for the last 5 minutes for the RH SSO API in the {{ $labels.namespace }} namespace
```
Comment (jessesarn) [inline deploy/template/prometheus-rule.yaml:98]: Shouldn't this say 1 second, not 10 seconds?

Comment (austincunningham) [reply deploy/template/prometheus-rule.yaml:98]: Good catch :+1:

**Review by jessesarn — APPROVED** [review]
LGTM

**Comment by austincunningham** [comment]
@jessesarn as the release merge blocker has been removed , do you have rights to merge this ?

**Comment by jessesarn** [comment]
Looks like I did ;)

**Pull request merged**